api.go

```diff
@@ -34,6 +34,7 @@ type PEFile struct {
 	VersionInformation map[string]string `json:"VersionInformation"`
 
 	Imports []string `json:"Imports"`
+	Exports []string `json:"exports"`
```

Comment:
For consistency should we name the field Exports ?

Reply:
Oops good catch! I'll fix shortly